plugins/rikki/heroeslounge/models/match/fields.yaml

```diff
@@ -1,40 +1,43 @@
 Tabs:
     fields:
-        wbp:
-            label: 'Schedule for'
+        tbp:
+            label: To be played before
             span: auto
             type: datepicker
+            disabled: true
             mode: datetime
             tab: General
         schedule_date:
-            label: 'Has to be scheduled until'
+            label: To be scheduled before
             span: auto
             type: datepicker
+            disabled: true
             mode: datetime
-            readOnly: true
             tab: General
-        tbp:
-            label: 'To be played until'
+        wbp:
+            label: Schedule for
             span: auto
             type: datepicker
             mode: datetime
-            readOnly: true
-            tab: General
-        round:
-            label: 'Round'
-            span: left
-            type: number
```

Comment:
Please allow round to be edited after creation - people sometimes make mistakes, this gives them the option to fix their mistake.